README.md

```diff
@@ -5,3 +5,19 @@ This is the sample Django application for the Azure Quickstart [Deploy a Python
 A Flask sample application is also available for the article at [https://github.com/Azure-Samples/msdocs-python-flask-webapp-quickstart](https://github.com/Azure-Samples/msdocs-python-flask-webapp-quickstart).
 
 If you need an Azure account, you can [create one for free](https://azure.microsoft.com/en-us/free/).
+
+## Generate SECRET_KEY
+
+Fill in a secret value in the `.env` file.
+
+For local development, use this random string as an appropriate value:
+
+```shell
+SECRET_KEY=123abc
+```
+
+For deployment to production, use this command to generate an appropriate value:
+
+```shell
+python -c 'import secrets; print(secrets.token_hex())'
```

Comment:
I don't think the .env file is going to be deployed, so this seems misleading? I think this only needs to be in the tutorial about deployment? (ie. between steps 3 and 4, unless you specify it in the command line)

Reply:
Okay, that's clear